### PR TITLE
fix: Updated step 12 for the CatPhotoApp

### LINKED ISSUE #47010: CatPhotoApp Step 12 is not clear
There are many users every day having issues with this step.

I have no idea what could be changed, but something would need to be changed

### PR DESCRIPTION
**Pull request by Jagrutiti**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47010 

Screenshot:
image

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ dashedName: step-12
 
 # --description--
 
-You can see the words `cat photos` underlined next to the image in the preview now. That's your link; feel free to give it a click. In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`. When you are done, remove the old anchor tag and text below the paragraph.
+In the previous steps you turned the words `cat photos` into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. In the preview you can see the words `cat photos` in blue and underlined next to the image. That's your link; feel free to give it a click.
+
+You can turn words inside of a `p` element into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`.
```
Comment (Sembauke) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md:12]:
```suggestion
You can turn words inside of a `p` element into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. Inside the `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ dashedName: step-12
 
 # --description--
 
-You can see the words `cat photos` underlined next to the image in the preview now. That's your link; feel free to give it a click. In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`. When you are done, remove the old anchor tag and text below the paragraph.
+In the previous steps you turned the words `cat photos` into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. In the preview you can see the words `cat photos` in blue and underlined next to the image. That's your link; feel free to give it a click.
+
+You can turn words inside of a `p` element into a link by surrounding them with an opening `<a>` tag and closing `</a>` tag. Inside the `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md:12]: This change clarifies how to nest the new `a` element, but completely omits the instruction to remove the *old* `a` element.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
I've added a comment here, but I'm not sure if this is the best approach as opposed to splitting the step up (as proposed in another PR).

**Comment by gikf** [comment]
Hey @Jagrutiti 

Thank you for opening this pull request.

As there was different PR changing the relevant step - #47320 -  merged recently, we are going to close this one.

Thank you and happy coding! 😊

---

If you have any questions, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [the contributors chat room](https://discord.gg/PRyKn3Vbay).

**Pull request closed without merging**